elements/pl-big-o-input/pl-big-o-input.py

```diff
@@ -130,6 +134,7 @@ def render(element_html: str, data: pl.QuestionData) -> str:
             "uuid": pl.get_uuid(),
             display.value: True,
             "show_placeholder": size >= PLACEHOLDER_TEXT_THRESHOLD,
```

Comment:
Should a custom placeholder override this check?

Reply:
Same for pl-symbolic-input.

Reply:
It may not be worth having this check at all, since placeholder text can be manually set to the empty string. How does pl-string-input do it?

Reply:
Good point. `pl-string-input` doesn't have a default placeholder, so this check is not necessary there. `pl-number-input` has a default placeholder and an option for `show-placeholder`, but no check based on size (I'd be in favour of deprecating that option in favour of a blank placeholder). Feel free to remove this check.

Reply:
Don't remove existing options, for backwards compatibility.

Reply:
As far as I can tell, there's no option for `show-placeholder` outside of this check in symbolic or big-o input (i.e. just setting the size). So the proposed change is just to remove the check entirely and always show whatever placeholder is given (which can be emptied out).

Reply:
Ah, ok, got it. I though you meant in other elements.

Reply:
Oh no, just these two.